hyp3_metadata/templates/insar/color_phase_png.xml.j2

```diff
@@ -12,15 +12,15 @@ The color phase browse image is generated from the wrapped interferogram and amp
 
 {#
 Begin idAbs READABLE - not to be included in the xml template
-This file is a browse image of the wrapped interferogram for a pair of Sentinel-1 {{ granule_type }} granules, processed using GAMMA software. The data is generated from the interferogram phase prior to unwrapping, and is displayed on a background of the backscatter intensity values to give topographic context to the deformation. The pixel spacing is resampled so that the output image is 2048 pixels wide.
+This file is a browse image of the wrapped interferogram for a pair of Sentinel-1 {{ granule_type }} granules, processed using GAMMA software. The image is generated from the interferogram phase difference values prior to unwrapping, so the values are constrained to a wrapped range of 2-pi, represented by a complete cycle through the color ramp (one "fringe"). The fringes are displayed on a background of the backscatter intensity values to give topographic context to the deformation. The pixel spacing is resampled so that the output image is 2048 pixels wide.
 
 Reference Scene: {{ reference_granule_name }}
 Secondary Scene: {{ secondary_granule_name }}
 Processing Date/Time: {{ processing_date.isoformat(timespec='seconds') }}
 
-Because the fringes are on a wrapped 2-pi scale rather than representing continuous values, a wrapped interferogram tends to be most useful as a color visualization. Fringes that are very close together indicate greater amounts of deformation than wider fringes. Each fringe (one cycle through the color scale) indicates line-of-sight displacement equivalent to half the sensor wavelength. The Sentinel-1 SAR sensor has a wavelength of 5.6 cm, so each fringe would represent a line-of-sight displacement of 2.8 cm.
+BBecause the values are on a wrapped 2-pi scale rather than representing continuous phase difference values, a wrapped interferogram tends to be most useful as a color visualization. Each fringe (one cycle through the color ramp) indicates line-of-sight displacement equivalent to half the sensor wavelength. The Sentinel-1 SAR sensor has a wavelength of 5.6 cm, so each fringe would represent a line-of-sight displacement of 2.8 cm. Fringes that are very close together indicate greater amounts of deformation than wider fringes.
```

Comment:
Typo
```suggestion
Because the values are on a wrapped 2-pi scale rather than representing continuous phase difference values, a wrapped interferogram tends to be most useful as a color visualization. Each fringe (one cycle through the color ramp) indicates line-of-sight displacement equivalent to half the sensor wavelength. The Sentinel-1 SAR sensor has a wavelength of 5.6 cm, so each fringe would represent a line-of-sight displacement of 2.8 cm. Fringes that are very close together indicate greater amounts of deformation than wider fringes.
```